### PR TITLE
BZ#2078813: Added CLI steps for StorageClass creation

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.9+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2078813

Link to docs preview: 
https://51344--docspreview.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-aws-efs.html#storage-create-storage-class_persistent-storage-csi-aws-efs

https://51344--docspreview.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-aws-efs.html#csi-dynamic-provisioning-aws-efs_persistent-storage-csi-aws-efs

QE review:
- [x] QE has approved this change.

Extension of PR #50108

### CODE REVIEW

File: modules/storage-create-storage-class-console.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
+:_content-type: PROCEDURE
+[id="storage-create-storage-class-console_{context}"]
+= Creating the {StorageClass} storage class using web console
```
Comment (kcarmichael08) [inline modules/storage-create-storage-class-console.adoc:3]: Suggestions: 

1. add "the" before console - "using the web console"
2. should "object" be added after storage class? (as in other comment)

**Review by kcarmichael08 — COMMENTED** [review]
Overall this looks really good! I had some minor comments & suggestions. 

Overall comment:

Modules & assemblies need an abstract tagged with [role="_abstract"] - for procedure module, see: https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc For assembly see: https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc

File: modules/storage-create-storage-class-cli.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+parameters:
+  provisioningMode: efs-ap <1>
+  fileSystemId: fs-a5324911 <2>
+  directoryPerms: "700" <3>
+  gidRangeStart: "1000" <4>
+  gidRangeEnd: "2000" <4>
+  basePath: "/dynamic_provisioning" <5>
+----
+<1> `provisioningMode` must be `efs-ap` to enable dynamic provisioning.
+<2> `fileSystemId` must be the ID of the EFS volume created manually above.
```
[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply modules/storage-create-storage-class-cli.adoc:25]: This is existing content which was restructured. Will incorporate this comment though. Thank you.

File: modules/storage-create-storage-class-cli.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<3> `directoryPerms` is the default permission of the root directory of the volume. In this case, the volume is accessible only by the owner.
```
[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply modules/storage-create-storage-class-cli.adoc:26]: Agreed. This is an existing content which was restructured. AFAI understand, it's the value mentioned in the example. Will reword.

File: modules/storage-create-storage-class-cli.adoc [hunk 4]
```diff
@@ -0,0 +1,35 @@
+:_content-type: PROCEDURE
+[id="storage-create-storage-class-cli_{context}"]
+= Creating the {StorageClass} storage class using CLI
```
[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply modules/storage-create-storage-class-cli.adoc:3]: `StorageClass` is a proper noun which will need a trailing `object`. storage class is a common noun which will not need `object`.

File: storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc [hunk 5]
```diff
@@ -41,11 +41,17 @@ include::modules/persistent-storage-csi-efs-sts.adoc[leveloffset=+1]
...
+include::modules/storage-create-storage-class-console.adoc[leveloffset=+2]
+include::modules/storage-create-storage-class-cli.adoc[leveloffset=+2]
 
 include::modules/persistent-storage-csi-efs-create-volume.adoc[leveloffset=+1]
 
 include::modules/persistent-storage-csi-dynamic-provisioning-aws-efs.adoc[leveloffset=+1]
 If you have problems setting up dynamic provisioning, see xref:../../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#efs-troubleshooting_persistent-storage-csi-aws-efs[AWS EFS troubleshooting].
+[role="_additional-resources"]
+.Additional resources
+* xref:../../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#efs-create-volume_persistent-storage-csi-aws-efs[Creating AWS EFS volume(s).]
```
[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc:53]: Good catch!

**Pull request merged**